lib/types.ts

```diff
@@ -157,6 +157,14 @@ export interface PluginOptions {
    */
   imageNameAndDigest: string;
 
+  /**
+   * WARNING! This is NOT used by the Snyk CLI!
+   *
+   * It is used by Docker Registry Agent to preserve the image digests if we can extract this
+   * information from pulling the image with Snyk Docker Pull.
+   */
+  digests: { manifest?: string; index?: string };
```

Comment:
I thought about asking to have some "complex" type to do something like this (pseudocode):
```
digests?: AtLeastOneOf{ manifest: string, index: string}
```
But the solution's actually quite complex, so...let's keep it the way it is 😆